src/python/grpcio/grpc/__init__.py

```diff
@@ -348,6 +348,82 @@ def details(self):
         raise NotImplementedError()
 
 
+##############  Invocation-Side Interceptor Interfaces & Classes  ##############
+
+
+class UnaryClientInfo(six.with_metaclass(abc.ABCMeta)):
```

Comment:
Good thought.

This can also be accomplished with **kwargs, but I'm not sure which is preferable.  I'll defer to @nathanielmanistaatgoogle on this one.